prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -108,6 +110,12 @@ modules_enabled = {
 		"s2sout_override";
 		"s2s_whitelist";
 		{{ end -}}
+
+		-- metrics collection functionality
```

Comment:
This should be inside the if statement, as otherwise it will be rendered in the final document no matter whether metrics are enabled or not.